libs/membership/website/src/lib/invoice-list/invoice-list-container.tsx

```diff
@@ -27,7 +27,7 @@ export function InvoiceListContainer({filter, className}: InvoiceListContainerPr
   } = useInvoicesQuery({
     onCompleted(data) {
```

Comment:
apparently onCompleted callbacks gets called on server side too when the data is there already